docs-kits/kits/Certificate Management Kit/page-changelog.md

```diff
@@ -5,13 +5,27 @@ description: 'Certificate Management Kit'
 sidebar_position: 1
 ---
 
-![Business partner kit banner](/img/Certificate_Kit_Icon.png)
+![Certificate Management kit banner](/img/kit-icons/certificate-kit-icon.svg)
 
 ### Certificate Management KIT
 
 All notable changes to this Kit will be documented in this file.
 
-## [0.1.0] - 2024-02-14
+## [2.0.0] - unreleased
```

Comment:
This should be 2.0.1 - please confirm @SujitMBRDI :)

Reply:
Hi @maximilianong, anyways this will be unreleased version for now and we will release with [2.0.0] at the end.
So, whatever changes @Unsharm will do that we can keep on adding in subsections like Added, Changed or Remove